configuration/configuration.go

```diff
@@ -247,6 +247,26 @@ type Configuration struct {
 			Classes []string `yaml:"classes"`
 		} `yaml:"repository,omitempty"`
 	} `yaml:"policy,omitempty"`
+
+	// OpenTelemetry configuration
+	OpenTelemetry *OpenTelemetryConfig `yaml:"otel,omitempty"`
+}
+
+// OpenTelemetryConfig provides open telemetry configuration
+type OpenTelemetryConfig struct {
+	// ServiceName is telemetry define one server name
+	ServiceName string `yaml:"serviceName,omitempty"`
+	// ExporterName must be otlp
+	ExporterName string `yaml:"exporterName"`
+	// ExporterEndpoint is otlp server addr
+	ExporterEndpoint string `yaml:"exporterEndpoint"`
```

Comment:
Would it make sense to add another level of config for exporter params?

There are other configuration that can be passed to the exporter. If you ever want to expose those, you would have to prefix all of them with `Exporter` which smells IMO.

Reply:
I think sticking `Exporter` params into a dedicated `struct` makes sense. @lengrongfu would you mind making that change?

Reply:
ping @lengrongfu